libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/RouteViewModel.java

```diff
@@ -120,15 +121,27 @@ public void fetchRouteFromOffRouteEvent(OffRouteEvent event) {
     }
   }
 
+  /**
+   * Sets the callback for reroute events and faster routes
+   *
+   * @param callback to set
+   */
+  public void setCallback(Callback callback) {
```

Comment:
Is there any situation in which we'd like to add more than one callback? For example in `NavigationViewModel` we're declaring a `List` of callbacks so wondering why not here as well. Am I missing something?

Reply:
Should we implement a way to remove callbacks? Is there any use case in which we'd like to remove a callback previously set?